dipy/reconst/dti.py

```diff
@@ -720,7 +720,7 @@ def __init__(self, gtab, fit_method="WLS", return_S0_hat=False, *args,
         is advisable to keep an eye on memory consumption as this value is
         increased.
 
-        Example : In :func:`iter_fit_tensor` we have a default step value of
+        e.g.: In :func:`iter_fit_tensor` we have a default step value of
```

Comment:
The phrase "e.g." should always be followed by a comma. And in this case, capitalized, as it's in the beginning of a sentence.